LibraBFT/Abstract/BFT.agda

```diff
@@ -540,14 +278,14 @@ module LibraBFT.Abstract.BFT
    --   - If CombinedPower (List-filter Meta-dishonest? xs ≡ CombinedPower xs we
    --   get a contradiction using the bft assumption (as we have now).
    find-honest : ∀ {xs : List Member}
-               → IsSorted _<Fin_ xs
+               → allDistinct xs
                → bizF + 1 ≤ CombinedPower xs
```

Comment:
This is (mathematically) equivalent to `bizF < CombinedPower xs`, which I think reads better and is more intuitive.  However, Agda doesn't treat them as the equivalent, so doing this would result in some changes to some proofs and would propagate to other properties as well.  I don't mind if you don't make this change, but it's something to consider.

Reply:
I did that. The proofs were very easy to fix.